dev-docs/hail-query-lifecycle.md

```diff
@@ -73,15 +73,8 @@ When invoked from Python, the JVM backends perform the following:
      CollectDistributedArray IR which represents the execution of a
      stage of the pipeline.  See `LowerTableIR`.
    - Shuffles (distributed sorts) are implemented on a per-backend
```

Comment:
Very interesting! Is there already a design doc?

Reply:
Notes from our first (only) meeting here:
https://gist.github.com/tpoterba/cd590b70ec303f23353b4dc4c7fdfa7d